src/Storefront/Framework/Seo/SeoUrlRoute/NavigationPageSeoUrlRoute.php

```diff
@@ -56,8 +56,9 @@ public function prepareCriteria(Criteria $criteria/*, SalesChannelEntity $salesC
     {
         $criteria->addFilter(new MultiFilter(MultiFilter::CONNECTION_AND, [
             new EqualsFilter('active', true),
-            new NotFilter(NotFilter::CONNECTION_AND, [
+            new NotFilter(NotFilter::CONNECTION_OR, [
```

Comment:
Could you add a testcase that now no SEO URLs are genereated for link (and also folder) categories inside the `\Shopware\Storefront\Test\Framework\Seo\NavigationPageSeoUrlTest::testGenerateForNewCategories()`